theme/static/js/hs_file_browser.js

```diff
@@ -139,16 +139,20 @@ function getFileTemplateInstance(file) {
     else {
         var title = hint + file.name + "&#13;" + fileTypeStr + "Size: " +
             formatBytes(parseInt(file.size));
-    }
-    
+    } 
+
     return "<li data-pk='" + file.pk + "' data-url='" + file.url + "' data-ref-url='" +
         file.reference_url + "' data-logical-file-id='" + file.logical_file_id +
         "' class='fb-file draggable' title='" + title + "' is-single-file-aggregation='" +
         file.is_single_file_aggregation + "' data-has-model-program-aggr-folder='" +
         file.has_model_program_aggr_folder + "' data-has-model-instance-aggr-folder='" +
         file.has_model_instance_aggr_folder + "' data-aggregation-appkey='" + file.aggregation_appkey  + "'>" +
         iconTemplate +
-        "<span class='fb-file-name'>" + file.name + "</span>" +
+        "<span class='fb-file-name'>" + (
+            file.name.length > 15
+                ? file.name.slice(0, 8) + "..." + file.name.slice(file.name.lastIndexOf('.') - 5)
+                : file.name
+        ) + "</span>"  +
```

Comment:
changing the `fb-file-name` in this way is going to break some functionality.
For example, it breaks the **rename** functionality here:
https://github.com/hydroshare/hydroshare/blob/9bde8ead436489afdee41dfd8d27a22ce3867144/theme/static/js/hs_file_browser.js#L2505

https://github.com/user-attachments/assets/f0bc47f7-d1a9-48fa-a279-c0b4ea39338c

Reply:
same issue still present as of your latest commit https://github.com/hydroshare/hydroshare/pull/5615/commits/c744a686873fb5309d08a8752e7079fb81becf51

![image](https://github.com/user-attachments/assets/8ce95516-be43-4eda-982b-437cae6cd34a)